File: DVLD_Project/GetAppInfo.cs
using Guna.UI2.WinForms;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DVLD_Project
{
    public partial class GetAppInfo : UserControl
    {
        public GetAppInfo()
        {
            InitializeComponent();
        }

        private void label5_Click(object sender, EventArgs e)
        {

        }

        private void pictureBox1_Click(object sender, EventArgs e)
        {

        }

        private void lblTypeID_Click(object sender, EventArgs e)
        {

        }

        private void groupBox1_Enter(object sender, EventArgs e)
        {

        }

        public void LoadInformations(int AppID, string ClassName, int NumOfExamsHasDone)
        {

            lblAppID.Text = AppID.ToString();
            lblClassName.Text = ClassName.ToString();
            lblExamsDone.Text = NumOfExamsHasDone.ToString() + "/3 - Has Done !..";
            if (NumOfExamsHasDone==3)
            {
                linkLabel1.Enabled = true;
            }
            else
            {
                linkLabel1.Enabled = false;
            }

        }

        private void GetAppInfo_Load(object sender, EventArgs e)
        {

        }
    }
}

File: DVLD_Project/GetAppInfo.vb
Imports System
Imports System.Windows.Forms

Namespace DVLD_Project
    Public Partial Class GetAppInfo
        Inherits UserControl
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub label5_Click(sender As Object, e As EventArgs)

        End Sub

        Private Sub pictureBox1_Click(sender As Object, e As EventArgs)

        End Sub

        Private Sub lblTypeID_Click(sender As Object, e As EventArgs)

        End Sub

        Private Sub groupBox1_Enter(sender As Object, e As EventArgs)

        End Sub

        Public Sub LoadInformations(AppID As Integer, ClassName As String, NumOfExamsHasDone As Integer)

            lblAppID.Text = AppID.ToString()
            lblClassName.Text = ClassName.ToString()
            lblExamsDone.Text = NumOfExamsHasDone.ToString() & "/3 - Has Done !.."
            If NumOfExamsHasDone = 3 Then
                linkLabel1.Enabled = True
            Else
                linkLabel1.Enabled = False
            End If

        End Sub

        Private Sub GetAppInfo_Load(sender As Object, e As EventArgs)

        End Sub
    End Class
End Namespace
